src/library_dylink.js

```diff
@@ -144,7 +144,7 @@ var LibraryDylink = {
   $GOTHandler__internal: true,
   $GOTHandler__deps: ['$GOT', '$currentModuleWeakSymbols'],
   $GOTHandler: {
-    'get': function(obj, symName) {
```

Comment:
same question in this file

Reply:
Good question.  At least for this use case we have plenty of test coverage I think.. perhaps not for the module splitting use case.  I will run some specific tests on on closure to see if this is needed.. perhaps creating a separate PR to remove the quotes.

Reply:
For whatever reason it looks like "get" is special cases by closure !  I  guess thats a good thing.   I put this in my `pre.js`:

```
var x = {
  get(b) { return 43 },
};

console.log(x);
```

When the method name is foo this outputs:

```{ g: [Function: g] }```

When the method name is `get` this outputs:

```{ get: [Function: get] }```

Wow.

Reply:
Weird, I wonder if that's because get/set are usually getters/setters for classes.

Reply:
I would imagine so yes... and because they are using by Proxy objects?   closure has a lot of hidden magic in there it seems.